diesel/Cargo.toml

```diff
@@ -27,5 +28,5 @@ default = ["postgres"]
 unstable = []
 large-tables = []
 huge-tables = ["large-tables"]
-postgres = ["pq-sys"]
+postgres = ["pq-sys", "uuid"]
```

Comment:
I'd prefer not to default to uuid. People can enable it explicitly if they want it.